mappings/net/minecraft/block/BlockState.mapping

```diff
@@ -6,8 +6,10 @@ CLASS net/minecraft/class_2680 net/minecraft/block/BlockState
 		FIELD field_16558 opaque Z
 		FIELD field_16559 DIRECTIONS [Lnet/minecraft/class_2350;
 		FIELD field_16560 shapes [Lnet/minecraft/class_265;
+		FIELD field_17651 oversized Z
```

Comment:
What does this mean? `exceedsFullCube`?

Reply:
yes

Reply:
@liach If that's what it means, I think naming it `exceedsFullCube` would make it clearer. I think we also have a siminarly named method in the VoxelShapes class.

Reply:
@liach You still haven't renamed this to `exceedsFullCube`. Did you just forget, or do you think `oversized` is better?

Reply:
I should revert it and leave it to #839 instead.

Reply:
That PR is very outdated (1.14.4), it will have many conflicts anyway. It would be nice to just have the name as soon as possible.

Reply:
Left this so that future digs in shape can better cover this.